### DISCUSSION TITLE
Which Key is missing some icons

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4010] [Q&A]
The latest Lazyvim does not look the same as the images in the readme.

https://github.com/folke/which-key.nvim

![image](https://github.com/user-attachments/assets/a94cae22-eeae-43f1-958f-84204d916c68)

@folke  What font do you use? I think I'm missing some icons.

![image](https://github.com/user-attachments/assets/69a79717-79f9-4ae0-8318-38535d37b34b)

![image](https://github.com/user-attachments/assets/b8a72d10-9292-41c0-a143-a8fa8be44906)

**Comment by dotfrag** [comment]
I have the same and all my fonts are up to date. Using a patched nerd font + symbols nerd font mono.

**Comment by folke** [comment]
Not every keymap has a defined icon yet.
The ones without icon show empty right now.
Maybe I should add a default icon?

**Reply by Old-Farmer** [reply]
I think a default icon is quite suitable for a consistent view of which-key pane.

**Comment by folke** [comment]
right, that's because I use fzf-lua and aded correct icons for that.
In the meantime, a PR has been merged to show icons for telescope as well

**Reply by folke** [reply]
Although, neo-tree icons should show.
Can you post a full screenshot?
Is mini.icons up to date?

**Reply by dotfrag** [reply]
I still don't see neo-tree (everything is up-to-date)

![image](https://github.com/user-attachments/assets/1b8168a6-25c0-4a09-88db-d4976c25eaac)

**Reply by dotfrag** [reply]
And stock lazyvim starter just in case:

![image](https://github.com/user-attachments/assets/23eaa4af-6604-4e49-b839-9031a663ef03)

**Reply by folke** [reply]
Damn, I forgot to make a new release of lazy.nvim.
Releasing now https://github.com/folke/lazy.nvim/pull/1634

After the update you'll see the missing icons.

Let me know if that indeed fixes it

**Reply by dotfrag** [reply]
Yes that worked! Thanks Folke!

**Reply by dotfrag** [reply]
Keywordprg is still empty, but I'm guessing that's because there's no icon for that.

**Comment by danielwe** [comment]
How about an info icon like `` for Keywordprg? (Doesn't render in the browser obviously, go to https://www.nerdfonts.com/cheat-sheet and search for `nf-seti-info` to see what I mean.)
